embeddings/utils/json_dict_persister.py

```diff
@@ -0,0 +1,13 @@
+import json
+from typing import Any, Dict
+
+from embeddings.utils.results_persister import ResultsPersister
+
+
+class JsonPersister(ResultsPersister[Dict[str, Any]]):
+    def __init__(self, path: str):
+        self.path = path
+
+    def persist(self, result: Dict[str, Any], **kwargs: Any) -> None:
+        with open(self.path, "w") as json_file:
+            json_file.write(json.dumps(result))
```

Comment:
```suggestion
            json.dump(result, json_file)
```

or just use `srsly`

Reply:
do we have it already? if we have then for sure, if not I am quite not sure if we should add it as it is not a crucial part, and we will not get any benefits beside increasing the size of the lib

Reply:
It was added for promises-election-dataset, but currently I think it is only used in experimental part. Srlsly has nice parser for jsonl format.

Reply:
As for now I would accept Albert's suggestion @riomus